test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/DictionaryObjectTypeVisitorTests.cs

```diff
@@ -187,5 +187,48 @@ public void Given_Type_When_PayloadVisit_Invoked_Then_It_Should_Return_Result(Ty
                 result.AdditionalProperties.Reference.Should().BeNull();
             }
         }
+
+        [TestMethod]
+        public void Given_Alias_Type_When_Visit_Invoked_Then_It_Should_Return_All_Sub_Schemas()
+        {
+            var originSchemaKey = "fakeAliasModel";
```

Comment:
Can we use `[DataTestMethod]` instead of `[TestMethod]` here so that we can avoid the `fakeAliasModel` value hard-coded everywhere?